native/src/components/PoisBottomSheet.tsx

```diff
@@ -86,24 +86,26 @@ const PoisBottomSheet = ({
 
   return (
     <StyledBottomSheet
+      accessibilityLabel=''
       index={snapPointIndex}
       isFullscreen={isFullscreen}
       snapPoints={snapPoints}
       enableContentPanningGesture={enableContentPanningGesture}
       enableDynamicSizing={false}
       animateOnMount
-      accessibilityRole={'list'}
-      accessibilityLabel={t('nearYou', { count: pois.length, nearYou: t('listTitle') })}
       handleComponent={BottomSheetHandle}
       onChange={setSnapPointIndex}>
       <BottomSheetContent>
         {slug ? (
-          <BottomSheetScrollView showsVerticalScrollIndicator={false}>{PoiDetail}</BottomSheetScrollView>
+          <BottomSheetScrollView accessibilityLabel={t('detailsInformation')} showsVerticalScrollIndicator={false}>
```

Comment:
:thinking: Not sure whether it would make more sense to set this to the title of the poi instead of `Information`? What do you think?

Reply:
Doesn't it make more sense to not have a label/have an empty label here then?